playbooks/roles/mongodb/tasks/main.yml

```diff
@@ -137,3 +148,9 @@
         dest: /etc/consul/mongodb-exporter.json
       notify:
         - reload consul
+
+    - name: Install pip, it is later needed for greendayonfire.mongodb
+      apt:
+        pkg:
+          - python-pip
+          - python3-pip
```

Comment:
You can remove these lines, they weren't required when running the full playbooks.

Reply:
Done. Finished.